central/apitoken/service/service_impl.go

```diff
@@ -3,6 +3,7 @@ package service
 import (
 	"context"
 	"strings"
+	"sort"
```

Comment:
```suggestion
	"slices"
```
(plus it'd have to go above `"strings"`